webofneeds/won-owner-webapp/src/main/webapp/jspm_config.js

```diff
@@ -13,7 +13,7 @@ System.config({
   },
 
   map: {
-    "Medium.js": "github:jakiestfu/Medium.js@1.0.1",
+    "Medium.js": "github:jakiestfu/Medium.js@master",
```

Comment:
shouldnt this version also be just the frozen on the commit number? see package.json